app/concepts/Payment/wrapper/representers/representer.rb

```diff
@@ -10,14 +10,12 @@ def initialize(wrapper)
         end
         
     
-        def seats_not_taken(id,cinema_hall_id, movie_id)
+        def seats_not_taken(params)
           return {"empty_seats":
           (
-            UseCase::CinemaHalls::GenerateSeats.new(
-            wrapper.left_Repository
-            ).call(cinema_hall_id) - UseCase::Tickets::Taken.new(
-            wrapper.right_Repository
-          ).call(id, cinema_hall_id, movie_id)
+            left=UseCase::CinemaHalls::GenerateSeats.new(wrapper.left_Repository).call(params[:cinema_hall_id]) 
+            right=UseCase::Tickets::Taken.new(wrapper.right_Repository).call(params)
```

Comment:
```suggestion
            taken_seats=UseCase::Tickets::Taken.new(wrapper.right_Repository).call(params)
```

Try to pick more meaningful names of the variables. It helps with reading the code.

Reply:
resolved